waspc/src/Wasp/Util.hs

```diff
@@ -118,3 +120,9 @@ insertAt :: [a] -> Int -> [a] -> [a]
 insertAt theInsert idx host =
   let (before, after) = splitAt idx host
    in before ++ theInsert ++ after
+
+-- TODO(shayne): should we be defining a new helper operator, or something else/better instead?
```

Comment:
Ha interesting! Well from what I have seen it works rather nicely in your code, so let's keep it? I don't have a better idea.

You could even go further and also define `<:> :: Monad m => m a -> m [a] -> m [a]`, I think you would find that useful in some places in your code.

Reply:
Yeah good point, we could consider adding that one too so we could get rid of `sequence [...] <++> ...` bits. Will think on it/test it out and see if it helps, good idea 👍🏻

Reply:
Used `<:>` in a few places, but felt many of the `sequence [...] <++> ... ` bits still were a bit cleaner. But will be helpful in future too, I think. 👍🏻